src/server_overrides.cpp

```diff
@@ -113,13 +115,26 @@ struct StubCookieAuthority : public mir::cookie::Authority
 
 void usc::server_overrides(mir::Server& server)
 {
-    server.wrap_cursor_listener([&server](std::shared_ptr<mir::input::CursorListener> const& default_)
+    bool enable_hardware_cursor{false};
```

Comment:
This patch introduces a segfault, for reasons that are obvious in retrospect: the `enable_hardware_cursor` variable will be out of scope by the time the callback passed to `wrap_cursor_listener` is invoked.  This patch should therefore be dropped for the time being (we can always come back to it later).